### DISCUSSION TITLE
Can I create a service account in kratos

### DISCUSSION THREAD

**Discussion by aravindakumarbootlabs** [ory/kratos#1825] [Q&A] [answered]
Is creating service account possible in kratos, if not how can I implement it from my side (is it even possible), how difficult is it to implement it in Kratos (if I want to do it). Can I please get some pointers?

**Comment by aeneasr** [comment, marked as answer]
You’re probably looking for OAuth2 Client Credentials grant which you can do with Ory Hydra!
